gym/utils/env_checker.py

```diff
@@ -126,53 +126,6 @@ def check_reset_seed(env: gym.Env):
         )
 
 
-def check_reset_info(env: gym.Env):
```

Comment:
Would we want to keep these with a signature check?

Reply:
Do you mean if "return_info" is in the signature, check to see if the return_info argument is handled correctly? So it would be for legacy third part envs implementing return_info in their signature?

Reply:
More like, if return info is in the signature then say that this feature is deprecate and the environment shouldn't use it

Reply:
done, I added a test for the signature check as well